master/reference/calicoctl/resources/hostendpoint.md

```diff
@@ -1,30 +1,30 @@
 ---
-title: Host Endpoint Resource (hostEndpoint)
+title: Host Endpoint Resource (HostEndpoint)
 ---
 
-A Host Endpoint resource (hostEndpoint) represents an interface attached to a host that is running Calico.
+A Host Endpoint resource (HostEndpoint) represents an interface attached to a host that is running Calico.
```

Comment:
I thought it would just be "host endpoint" and \`HostEndpoint\`- did @emanic give this a thumbs up?

Reply:
hmm it was like that previously for all the definitions, but I can change it to that